enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/assets/DependencyUtils.java

```diff
@@ -64,4 +56,39 @@ public static void addDependency(Project project, String groupId, String artifac
         }
     }
     
+    public static void addAnnotationProcessor(Project project, String groupId, String artifactId) {
+        Project projectToModify = getProjectToModify(project, "lib");
+        
+        if (projectToModify != null) {
+            ArtifactSpec spec = ArtifactSpec.make(groupId, artifactId);
+            Dependency dep = Dependency.make(spec, Scopes.PROCESS);
+            DependencyChange change = DependencyChange.builder(DependencyChange.Kind.ADD)
+                    .dependency(dep)
+                    .create();
+
+            try {
+                ModificationResult mod = ProjectDependencies
+                        .modifyDependencies(projectToModify, new DependencyChangeRequest(Collections.singletonList(change)));
+                mod.commit();
+            } catch (IOException | DependencyChangeException | ProjectOperationException ex) {
+                throw new IllegalStateException(ex);
```

Comment:
I am not sure with the handling here. IOException and ProjectOperationException are generic failures (no chance to handle, IMHO), but `DependencyChangeException` indicates that perhaps the dependency is already there or other 'recoverable' failure (?)

Reply:
If dependency is already there we should silently continue IMO, and for other failures, I am not sure if we can recover

Reply:
In that case, use `.option(Options.skipConflicts)`